src/node/txreconciliation.h

```diff
@@ -57,6 +57,14 @@ class TxReconciliationTracker {
      * This function must be called only once per peer.
      */
     std::tuple<bool, bool, uint32_t, uint64_t> PreRegisterPeer(NodeId peer_id, bool peer_inbound);
+
+    // Helpers
+
+    /**
+     * Attempts to forget reconciliation-related state of the peer (if we previously stored any).
+     * After this, we won't be able to reconcile with the peer.
```

Comment:
With the [latest code](https://github.com/bitcoin/bitcoin/blob/03d76fa4385bfac2a58972ed3caaca1de788b2bb/src/net_processing.cpp#L2807), we also call it when a peer forgot to send us wtxid (although being pre-registered by sending us SENDRECON). So `DisconenctedPeer()` would be quite inaccurate.

Reply:
Makes sense - can you update the commit message of 182d984136066fd4f09b3e1a1bbd9b50c6cb87be to say "Forget peer's reconciliation state on disconnect and when wtxidrelay is not sent" so that it's documented there as well?